documentation/modules/configuring/ref-storage-persistent.adoc

```diff
@@ -5,36 +5,48 @@
 [id='ref-persistent-storage-{context}']
 = Persistent storage
 
-Persistent storage uses {K8sPersistentVolumeClaims} to provision persistent volumes for storing data.
-Persistent Volume Claims can be used to provision volumes of many different types, depending on the {K8SStorageClass} which will provision the volume.
-The data types which can be used with persistent volume claims include many types of SAN storage as well as {K8sLocalPersistentVolumes}.
+[role="_abstract"]
+Persistent data storage retains data even when a pod that uses the storage restarts.
```

Comment:
It should not keep them just between restarts but make them persisted in general, survive all kind of failures and disasters etc.. So maybe we should make this statement more wide than just pod restarts?

Reply:
changed to:

> Persistent data storage retains data in the event of system disruption.
> For pods that use persistent data storage, data is persisted across failures and restarts.